dash/components/src/Users/Activity/ActivityItemBtns.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import cx from 'classnames';
+import { Button } from '@shared/components';
+import ActivityItemTime from './ActivityItemTime';
+
+interface Props {
+  className?: string;
+  flagged: boolean;
+  date: Date;
+  onApprove(): unknown;
+  onFlag(): unknown;
+}
+
+const ActivityItemBtns: React.FC<Props> = ({
+  className,
+  onFlag,
+  onApprove,
+  flagged,
+  date,
+}) => (
+  <div className={cx(`flex justify-between items-center`, className)}>
+    <ActivityItemTime date={date} />
+    <div className="flex gap-2">
+      <div
+        data-tooltip-id="flag-activity"
+        data-tooltip-content={
+          flagged
+            ? `Flagged: won’t be deleted until unflagged`
+            : `Flag to prevent deletion for 60 days`
```

Comment:
added a tool tip for a little discoverability/self-documentation. i plan to do a little blog post and maybe a dashboard "announcement", but want to get the feature in first, and maybe use it a little personally